webpack.config.js

```diff
@@ -0,0 +1,28 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  mode: "development",    
```

Comment:
```suggestion
  mode: process.env.NODE_ENV,    
```
Add please create env folder and  .env.development &  .env.production files which should includes env variables

P.S. use dotenv npm package

```
const env = require('dotenv');

env.config({
  path: `${__dirname}/env/.env.${process.env.NODE_ENV ? process.env.NODE_ENV : 'development'}`,
});```

Reply:
**out of scope!**

it should be implemented in other PR

Reply:
Okay, understand